packages/react-badge/src/Badge/Badge.types.ts

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { ComponentProps } from '@fluentui/react-utils';
+import { SizeValue } from '@fluentui/theme';
```

Comment:
```suggestion
```

Oops, we can't have dependency on this package as it's not converged

Reply:
What should we use for SizeValue then?